app.json

```diff
@@ -7,7 +7,6 @@
     "description": "Travellers community for sharing, hosting and getting people together.",
     "githubUrl": "https://github.com/Trustroots/trustroots-expo-mobile",
     "privacy": "public",
-    "sdkVersion": "37.0.0",
```

Comment:
I now get error `Invalid sdkVersion.` on `npm start` — what was the rationale for removing this? 🤔

Reply:
My bad! — I had expo cli 2.x and 3.x cleaned this out. :-)